config/basicsettings.cpp

```diff
@@ -41,6 +41,8 @@ BasicSettings::BasicSettings(LXQt::Settings* settings, QWidget *parent) :
 
     restoreSettings();
 
+    connect(spacingBox, QOverload<int>::of(&QSpinBox::valueChanged), this, &BasicSettings::save);
+    connect(widthBox, QOverload<int>::of(&QSpinBox::valueChanged), this, &BasicSettings::save);
```

Comment:
The changes seem good, but now that you are at it, please also fix two things that existed before your patch:

 1. Replace all instances of `QOverload<int>::of(&QSpinBox::valueChanged)` with `&QAbstractSpinBox::editingFinished` (I think there are 4). Writing to config file continously while a spinbox is changed by mouse wheel isn't good practice (not your fault, of course).
 2. The names `widthLabel` and `spacingLabel` should be swapped.

Reply:
Done both, but with the swapping I'm not sure, I replaced all of them, please recheck.